src/node/mempool_args.cpp

```diff
@@ -25,6 +25,9 @@ using common::AmountErrMsg;
 using kernel::MemPoolLimits;
 using kernel::MemPoolOptions;
 
+//! Maximum mempool size on 32-bit systems.
+static constexpr int MAX_32BIT_MEMPOOL_MB{500};
```

Comment:
ea561779abfb2298a9f20ceafa9e05fe2c58d841: let's just use 4096? This setting doesn't check actual memory capacity on 64-bit systems either, so we don't need to be this protective.

Reply:
... and I think it should be a `size_t` instead

Reply:
If it's set to the maximum RAM capacity on 32-bit system the process will start and then just randomly crash once there is high transaction traffic. Really if someone wants to use a large mempool they should use a 64-bit system.

I don't think this constant should be a `size_t` because it gets compared to a signed integer.